cmd/baton-google-workspace/main.go

```diff
@@ -40,9 +40,16 @@ func main() {
 func getConnector(ctx context.Context, cfg *config) (types.ConnectorServer, error) {
 	l := ctxzap.Extract(ctx)
 
-	jsonCredentials, err := os.ReadFile(cfg.CredentialsJSONFilePath) // just pass the file name
-	if err != nil {
-		l.Error("error reading crentialsJson file", zap.Error(err))
+	var jsonCredentials []byte
+
+	if cfg.CredentialsJSONFilePath == "" {
+		l.Error("no path specified to credentialsJson file")
```

Comment:
why dont we return an error here?

Reply:
Because one day we'll want to get connector capabilities to work without credentials. For now it just errors trying to create the client.